isthmus/src/main/java/io/substrait/isthmus/SubstraitRelVisitor.java

```diff
@@ -271,6 +271,7 @@ Aggregate.Measure fromAggCall(RelNode input, Type.Struct inputType, AggregateCal
     if (call.filterArg != -1) {
       builder.preMeasureFilter(FieldReference.newRootStructReference(call.filterArg, inputType));
     }
+    // TODO: handle the collation on the AggregateCall
```

Comment:
Filed an issues for this: https://github.com/substrait-io/substrait-java/issues/215